JavaScript/modal.js

```diff
@@ -0,0 +1,134 @@
+      highlight: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+      description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+      lang: ['html', 'css', 'javascript'],
+      live: 'https://eserengo.github.io/Portfolio-setup/',
+      source: 'https://github.com/eserengo/Portfolio-setup',
+    },
+  ];
+
+  const projectCardString = projectsData.map((project) => `
+    <div class="card">
+      <div class="inner-card">
+        <div class="card-img card-1">
+          <img
+            src="${project.src}"
+            alt="Project Snapshoot"
+          />
+        </div>
+        <div class="desc">
+          <h3>${project.title}</h3>
+          <h4>${project.subtitle} <span>${project.techStack}</span> <span>${project.year}</span></h4>
+          <p>${project.highlight}</p>
+          <ul>
+            ${project.lang.map((tech) => `<li>${tech}</li>`).join('')}
+          </ul>
+          <button type="button" class="project-btn button">
+            See Project
+          </button>
+        </div>
+      </div>
+    </div>
+  `);
+
+  const popupContainer = document.querySelector('.project-popup-container');
+  const recentWork = document.querySelector('.recent-work');
+  const parser = new DOMParser();
+
+  projectCardString.forEach((projectString, index) => {
+    const projectElement = parser.parseFromString(projectString, 'text/html').body.firstChild;
+
+    const mobilePopup = `
+      <div class="project-popup-hidden ">
+        <div class="project-background"></div>
+          <div class="project-popup">
+            <div class="project-header">
+              <i class="fa-solid fa-xmark project-close"></i>
+              <h2>${projectsData[index].title}</h2>
+              <p>${projectsData[index].subtitle} <span>${projectsData[index].techStack}</span> <span>${projectsData[index].year}</span></p>
+            </div>
+  
+            <div class="project-img">
+            <img
+            src="${projectsData[index].src}"
+            alt="Project Snapshoot"
+            /> 
+            </div>
+  
+            <div class="project-desc">
+              <p>${projectsData[index].description}</p>
+              <ul>
+                ${projectsData[index].lang.map((tech) => `<li>${tech}</li>`).join('')}
+              </ul>
+            </div>
+  
+            <div class="project-links">
+              <a href="${projectsData[index].live}">See Live <i class="fa-solid fa-arrow-up-right-from-square"></i></a>
+              <a href="${projectsData[index].source}">See Source <i class="fa-brands fa-github"></i></a>
+            </div>
+        </div>
+      </div>
+    `;
```

Comment:
Great job Generating the modal popup :clap: I have realized the following key changes that should be made:

- [ ] When I click the button to start the popup, I realize that I am not able to close the popup once it is opened. 
- [ ] I am also not able to view the rest of the popup when it is open kindly look into this. Below I have included video cast of how I can see it right now.

**This is how I am seeing it on desktop view:**

[Screencast from 07-09-2022 06:13:46 ALASIRI.webm](https://user-images.githubusercontent.com/86318284/188915923-9b1a38e6-938f-4174-8e52-f22cca3665d5.webm)

**This is how I am seeing it when I view it on mobile view:**

[Screencast from 07-09-2022 06:15:38 ALASIRI.webm](https://user-images.githubusercontent.com/86318284/188916299-8a9dea47-32c1-4c0e-a9f1-999387348ca7.webm)